shard.go

```diff
@@ -145,8 +156,10 @@ func (s *Shard) writeSeries(batch []byte) error {
 	})
 }
 
-func (s *Shard) deleteSeries(name string) error {
-	panic("not yet implemented") // TODO
+func (s *Shard) dropSeries(seriesID uint32) error {
+	return s.store.Update(func(tx *bolt.Tx) error {
```

Comment:
You should check first that `s.store != nil`. If this shard isn't local to the server, it will be, causing an error here.